### DISCUSSION TITLE
Hot reload support

### DISCUSSION THREAD

**Discussion by JoacimWall** [CommunityToolkit/Maui.Markup#23] [General]
Hi 
Tested Markup on windows and mac in Visual studio 2022.
Is it supposed to work with Hot restart/reload?    
 Is there any guide lines on how to be able to live code like Hot reload/restart?
can you use Reloadify3000? 

//regards Joacim

**Comment by danielftz** [comment]
> Hi Tested Markup on windows and mac in Visual studio 2022. Is it supposed to work with Hot restart/reload? Is there any guide lines on how to be able to live code like Hot reload/restart? can you use Reloadify3000?
> 
> //regards Joacim

From my experience, the hot reload has been inconsistent with MCT markup (most of the time not working at all). I think if you declare the maui component in an instance field first, then hot reload would work.

**Comment by TheCodeTraveler** [comment]
Closing this to consolidate the Hot Reload discussions here: https://github.com/CommunityToolkit/Maui.Markup/discussions/71
